bin/node-template/runtime/darwinia_types.json

```diff
@@ -118,17 +117,11 @@
 		"seal": "Vec<Bytes>",
 		"hash": "Option<H256>"
 	},
-	"Bloom": {
-		"_struct": "[u8; 256]"
-	},
-	"H128": {
-		"_struct": "[u8; 16]"
-	},
-	"H512": {
-		"_struct": "[u8; 64]"
-	},
+	"EthAddress": "H160",
+	"Bloom": "[u8; 256; Bloom]",
+	"H128": "[u8; 16; H128]",
```

Comment:
Does there any document about this syntax, does it mean some fallback/overwrite mechanism?